src/Umbraco.Core/Persistence/Repositories/RedirectUrlRepository.cs

```diff
@@ -185,7 +185,19 @@ public IEnumerable<IRedirectUrl> GetAllUrls(long pageIndex, int pageSize, out lo
         public IEnumerable<IRedirectUrl> GetAllUrls(int rootContentId, long pageIndex, int pageSize, out long total)
         {
             var sql = GetBaseQuery(false)
-                .Where("umbracoNode.path LIKE @path", new { path = "%," + rootContentId + ",%" })
+                .Where("[umbracoNode].[path] LIKE @path", new { path = "%," + rootContentId + ",%" })
```

Comment:
Cannot use square bracket syntax, this is for MSSQL only and will not work on MySql.  You need to use the SqlSyntax to get the correct quoted values like:  

`SqlSyntax.GetQuotedTableName("umbracoNode")` 
and
`SqlSyntax.GetQuotedColumnName("path")`